features/login_github_redirect.feature

```diff
@@ -0,0 +1,151 @@
+Feature: after a user logins with their github id they are redirected to the page they wanted to view prior to login
+
+  As a user who first opens the app
+  So that I can efficiently login and browse the app
+  I want to return to the page I requested to login
+
+Background: users, orgs and apps have been added to database
+  Given the following apps exist:
+        | name  | description | org_id | status  |
+        | app1  | test        | 1      | pending |
+        | app2  | test        | 1      | pending |
+        | app3  | test        | 1      | pending |
+
+    And the following orgs exist:
+        | name | contact_id |
+        | org1 | 1          |
+        | org2 | 1          |
+        | org3 | 1          |
+
+    And the following users exist:
+        | name  | github_uid      | email         |
+        | user1 | esaas_developer | test@user.com |
+        | user2 |                 | test@user.com |
+        | user3 |                 | test@user.com |
+
+
+  And I am not logged in
+
+Scenario: not logged in so should see login with Github from users page
```

Comment:
The three of the following scenario seems to test login_controller, not the redirect feature. If you intended to test the login_controller behavior, make a separate file for that.

Reply:
Agree with Sean!

Reply:
Thanks! Good point. I am moving them to a new file.